rpc-client-api/src/response.rs

```diff
@@ -294,10 +294,24 @@ pub struct RpcContactInfo {
     pub pubkey: String,
     /// Gossip port
     pub gossip: Option<SocketAddr>,
+    /// Tvu UDP port
+    pub tvu: Option<SocketAddr>,
+    /// Tvu QUIC port
+    pub tvu_quic: Option<SocketAddr>,
```

Comment:
I don't see much value in exposing this for now, as TVU over QUIC is not rolled out yet.